src/components/input/Select.tsx

```diff
@@ -355,8 +355,7 @@ function SelectMain<T>({
   listboxOverflow = 'truncate',
   'aria-label': ariaLabel,
   'aria-labelledby': ariaLabelledBy,
-  /* eslint-disable-next-line no-prototype-builtins */
-  listboxAsPopover = HTMLElement.prototype.hasOwnProperty('popover'),
+  listboxAsPopover,
```

Comment:
This was duplicated now. The fallback logic now lives in `Popover` and we don't need to define a default value here anymore.